nftables.go

```diff
@@ -136,34 +141,42 @@ func (nft *realNFTables) NewTransaction() *Transaction {
 
 // Run is part of Interface
 func (nft *realNFTables) Run(ctx context.Context, tx *Transaction) error {
+	nft.mu.Lock()
+	defer nft.mu.Unlock()
```

Comment:
So this creates a new API constraint; you can only have a single Run or Check running at a time... but I guess I can't really think of any reason you'd need to do two in parallel...

Since the mutex is only used to control access to `.buffer`, it might make more sense to call it `bufferMutex` or something.